Code/Source/Lidar/PipelineGraph.cpp

```diff
@@ -178,15 +177,35 @@ namespace RGL
     bool PipelineGraph::GetResults(RaycastResults& results)
     {
         int32_t resultSize = -1;
-        RGL_CHECK(rgl_graph_get_result_size(m_nodes.m_pointsFormat, rgl_field_t::RGL_FIELD_DYNAMIC_FORMAT, &resultSize, nullptr));
+        RGL_CHECK(rgl_graph_get_result_size(m_nodes.m_pointsYield, rgl_field_t::RGL_FIELD_DYNAMIC_FORMAT, &resultSize, nullptr));
```

Comment:
You cannot get the field `RGL_FIELD_DYNAMIC_FORMAT` because the yield node doesn't have it. In this case, you should check the size for each field separately.